lib/rex/logging/log_dispatcher.rb

```diff
@@ -118,24 +118,47 @@ def get_level(src)
 end
 end
 
-###
-#
-# An instance of the log dispatcher exists in the global namespace, along
-# with stubs for many of the common logging methods.  Various sources can
-# register themselves as a log sink such that logs can be directed at
-# various targets depending on where they're sourced from.  By doing it
-# this way, things like sessions can use the global logging stubs and
-# still be directed at the correct log file.
-#
-###
-ExceptionCallStack = "__EXCEPTCALLSTACK__"
+BACKTRACE_LOG_LEVEL = 3 # Equal to LEV_3
+DEFAULT_LOG_LEVEL = 3 # Equal to LEV_3
```

Comment:
These both seem a little high. `LEV_3` has a lot of other messaging that some people would find annoying (like the debug output of matching payloads together).

Reply:
`DEFAULT_LOG_LEVEL` set to 0